test/unitTest/java/org/thoughtcrime/securesms/database/ConversationSearchableLoaderTest.java

```diff
@@ -0,0 +1,8 @@
+package org.thoughtcrime.securesms.database;
+
+/**
+ * Created by Claudia on 2018-03-16.
+ */
+
+public class ConversationSearchableLoaderTest {
```

Comment:
Missing content for this test. Was it pushed accidentally? @claudds

Reply:
Implementation was pushed to a separate branch for Jacob to look at before merging to this one